### DISCUSSION TITLE
Failed to fetch some objects using Git LFS

### DISCUSSION THREAD

**Discussion by pppaw** [foxglove/community#828] [General]
Hello,

I have cloned the Foxglove repo to create a local Foxglove studio. After that, I tried to fetch objects using git lfs by using the command git lfs pull to ensure Git LFS objects are up-to-date but I am getting an error as follows: Failed to fetch some objects from 'https://github.com/foxglove/studio.git/info/lfs'. How do I solve this error?

**Comment by defunctzombie** [comment]
Probably a network error with your setup. We cannot reproduce this error unfortunately.
